en/editor/publish/build-open-data-context.md

```diff
@@ -0,0 +1,113 @@
+# Building Open Data Context Project
+
+## Introduction to Open Data Context
+
+At present, **WeChat**, **Baidu** and **ByteDance** are small game platforms that add the concept of **Open Data Context**, which is a separate game execution environment, in order to protect their social relationship chain data. The resources, engines, and applications in the Open Data Context are completely isolated from the main game, and only in the Open Data Context can developers access the relationship chain data through the open interface provided by the platform to implement some features such as ranked version.
+
+In Cocos Creator 3D, we have deprecated the maintenance of the Canvas Renderer module. The Open Data Context now only supports Canvas rendering.
+In the previous scheme, we used Cocos Creator 2D to build Open Data Context projects. As of v3.0, we've deprecated this practice and provided with a built-in Canvas engine. The engine is provided by the WeChat team and is a front-end lightweight Canvas engine based on XML+CSS design.
```

Comment:
In the previous scheme, we used Cocos Creator 2D to build Open Data Context projects. As of v3.0, we've deprecated this practice and provided with a built-in Canvas engine. The engine is provided by the WeChat team and is a front-end lightweight Canvas engine based on XML+CSS design.
-->
In the previous scheme, **Cocos Creator 2D** was used to build **Open Data Context** projects. As of **v3.0**, this practice has been deprecated and a built-in Canvas engine has been provided. The engine is provided by the WeChat team and is a front-end lightweight Canvas engine based on XML+CSS design.